spec/services/export_offenders_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe ExportOffenders do
+  describe '#call' do
+    let(:file) { Rails.root.join('tmp', 'soi_extract.csv') }
+
+    let!(:offender_1) do
```

Comment:
We use UUIDs so while it is doable to have factories with progressive numbers, it would add some complexity to manage these UUIDs in the right format.